controllers/nnf_workflow_controller.go

```diff
@@ -884,9 +884,8 @@ func (r *NnfWorkflowReconciler) finishPreRunState(ctx context.Context, workflow
 	case "persistentdw":
 		envName = "DW_PERSISTENT_" + dwArgs["name"]
 	case "container":
-		// TODO: set env variables for containers; job names? ports? container storage args?
-		// TODO: Check to see that jobs and pods have started. Example: pod volumes aren't ready -
-		// we shouldn't go ready in that case
+		return r.checkIfContainerJobsStarted(ctx, workflow)
```

Comment:
Consider using "waitFor..." to match other functions that do similar. i.e. `waitForContainersToStart()`